Dockerfile.worker.dev

```
@@ -0,0 +1,13 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.19
+USER 0
+ENV PROJECT_DIR=/backend \
+    CGO_ENABLED=0
+
+WORKDIR /backend
+RUN mkdir "/build"
+
+COPY . .
+RUN go install github.com/githubnemo/CompileDaemon@latest
+RUN make prep GO=go
+
+ENTRYPOINT ["bash", "-c", "go run ./cmd/pbackend worker && $HOME/go/bin/CompileDaemon -build='go build -buildvcs=false -o /build/pbackend ./cmd/pbackend' -command='/build/pbackend worker'"]
```

you can use just the `CompileDaemon `
```suggestion
ENTRYPOINT ["bash", "-c",  "$HOME/go/bin/CompileDaemon -build='go build -buildvcs=false -o /build/pbackend ./cmd/pbackend' -command='/build/pbackend workeri'"]
```